.travis.yml

```diff
@@ -11,7 +11,7 @@ cache:
   - output/
 
 before_install:
-  - curl -Ls -o docker-build https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/docker-build
+  - curl -Ls -o docker-build https://github.com/mate-desktop/mate-dev-scripts/raw/travis/travis/docker-build
```

Comment:
it doesn't need master branch?

Reply:
Yes, need master branch, but there has a PR block this. https://github.com/mate-desktop/mate-dev-scripts/pull/16

Reply:
ah ok, so, needs to be merged other PR first?